utils/src/server/rpc_module.rs

```diff
@@ -506,7 +487,7 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 		Ok(())
 	}
 
-	/// Register an `alias` name for an `existing_method`.
+	/// Register an alias for an existing_method. Alias uniqueness is enforced at compile time.
```

Comment:
This is only true for the proc macros and not in this context but for most use-cases your comment is true :P

```suggestion
	/// Register an alias for an existing_method.
```

Reply:
Would `Alias uniqueness is enforced.` be correct?

Reply:
Yes, a duplicated alias will return error